qoqo/README_qoqo.md

```diff
@@ -1,65 +1,101 @@
 
 ```shell
-# can be necessary on macOS
-pip install qoqo
+# can be necessary on mscOS
+RUSTFLAGS="-C link-arg=-undefined -C link-arg=dynamic_lookup" pip install qoqo
 ```
 
+```shell
+RUSTFLAGS="-C link-arg=-undefined -C link-arg=dynamic_lookup" maturin build -m qoqo/Cargo.toml  --release
+pip install target/wheels/$NAME_OF_WHEEL
+```
+
+### Using in a rust project
+
 When using qoqo in a rust project providing a python interface add
 
 ```TOML
 qoqo = {version="1.0.0", default-features=false}
 ```
 
-to the `[dependencies]` section of the project Cargo.toml.
\ No newline at end of file
+to the `[dependencies]` section of the project Cargo.toml.
+
+
+## Create your first circuit in qoqo
+
+The following code snippet can be used to create your first circuit in qoqo. For an expanded collection of **examples** please see the jupyter notebooks in the extra repository [qoqo_examples](https://github.com/HQSquantumsimulations/qoqo_examples). 
+
+```python
+from qoqo import Circuit
+from qoqo import operations as ops
+from qoqo_quest import Backend
+
+circuit = Circuit()
+circuit += ops.DefinitionBit(name="classical_reg", length=2, is_output=True)
+circuit += ops.PauliX(qubit=0)
+circuit += ops.CNOT(0,1)
+circuit += ops.PragmaDamping(qubit=0, gate_time=0.1, rate=0.1)
+circuit += ops.PragmaDamping(qubit=1, gate_time=0.1, rate=0.1)
+circuit += ops.MeasureQubit(qubit=0, readout="classical_reg", readout_index=0)
+circuit += ops.MeasureQubit(qubit=1, readout="classical_reg", readout_index=1)
+circuit += ops.PragmaSetNumberOfMeasurements(number_measurements=1000, readout="classical_reg")
+
+backend = Backend(number_qubits=2)
+bit_registers, float_registers, complex_registers = backend.run_circuit(circuit)
+```
+
+## Features
+
+**qoqo** provides the following functionalities and features:
+
+* A `Circuit` class to represent quantum circuits
+* A `QuantumProgram` class to represent quantum programs 
+* Classes representing single-qubit, two-qubit, multi-qubit and measurement operations that can be executed (decomposed) on any universal quantum computer
+* Classes representing so-called PRAGMA operations that only apply to certain hardware, simulators or annotate circuits with additional information
+* Support for symbolic variables
+* Readout based on classical registers
+* Measurement classes for evaluating observable measurements based on raw readout date returned by quantum computer backends
+* Serialization to json and deserialization from json for circuits and measurement information. Serialization support can easily be expanded to other targets with the help of the serde crate.
+
+## Support
+
+This project has been partly supported by [PlanQK](https://planqk.de) and is partially supported by [QSolid](https://www.q-solid.de/) and [PhoQuant](https://www.quantentechnologien.de/forschung/foerderung/quantencomputer-demonstrationsaufbauten/phoquant.html).
```

Comment:
This file doesn't have a Citation section, is this intentional?

Reply:
I will add it.